test/core/ExamplesUnitTests/TribitsExampleProject2_Tests.cmake

```diff
@@ -197,24 +197,24 @@ function(TribitsExampleProject2_find_tpl_parts  sharedOrStatic  findingTplsMetho
         "Tpl1_LIBRARY_NAMES='tpl1'"
         "Found lib '${tplInstallBaseDir}/install_tpl1/lib/libtpl1${libextregex}'"
         "TPL_Tpl1_LIBRARIES='${tplInstallBaseDir}/install_tpl1/lib/libtpl1${libextregex}'"
-        "Found header '${tplInstallBaseDir}/install_tpl1/include/Tpl1.hpp'"
+        "Found header '${tplInstallBaseDir}/install_tpl1/include[/]*/Tpl1.hpp'"
```

Comment:
Could you please give me a minimum `CMakeLists.txt` that reproduces the duplicate slash issue? This looks like a CMake bug to me, and I'd like to investigate. I tried a minimal `find_file(STDIO_H stdio.h)` and it didn't insert a duplicate slash.

Ignoring the broader issue, at the basic regex level, you could use `/+` instead of `[/]*/`.

Reply:
> Could you please give me a minimum CMakeLists.txt that reproduces the duplicate slash issue? This looks like a CMake bug to me, and I'd like to investigate. I tried a minimal find_file(STDIO_H stdio.h) and it didn't insert a duplicate slash.

Not sure how long that would take to create a minimal reproducer.  Is it not possible to just build a debug version of CMake and then directly debug the generation of this output?  You could even put in some debug print statements in the CMake C++ code to see what is happening.  That is likely the fastest way to track this down.

> Ignoring the broader issue, at the basic regex level, you could use `/+` instead of `[/]*/`.

Well, I would like to specifically like to match one or two `/` but not more than two.  But I guess you could use `/*/` and and that should work.

Reply:
`*` means "zero or more", not "zero or one". `/+` is the same thing as `[/]*/`. If you want "one or two", use `/?/` or `//?`.

Reply:
Addressed in PR #567.